ui/dashboards/src/components/DashboardToolbar/DashboardToolbar.tsx

```diff
@@ -65,7 +83,7 @@ export const DashboardToolbar = (props: DashboardToolbarProps) => {
                   Dashboard managed via code only. Download JSON and commit changes to save.
                 </Alert>
               )}
-              <Button variant="contained" onClick={onSave} disabled={isReadonly}>
+              <Button variant="contained" onClick={onSaveButtonClick} disabled={isReadonly || isSaveDashboard}>
```

Comment:
should we hide `Save` button if `onSave` is undefined?

Reply:
> should we hide the save button if `onSave` is undefined?

mmm that's an interesting question. But does it make sense then that you can still edit the dashboard.
I think for the moment we should keep the `save` button and at some point we should revisit the edit/save flow to be sure it's consistent depending of the different context (readonly UI, onSave not defined ...etc)

Reply:
> I wish MUI button had a `loading` prop :( and not this [experimental one ](https://mui.com/material-ui/api/loading-button/)

yeah that would be great, it would simplify so much work. We could still depend on `@mui/lab` if it's a wish we all share.